backend/classify.get.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+include_once 'classify.functions.php';
+
+if (!empty($_GET['IP'])) {
```

Comment:
This entire _file_ is a debug code, but my concern is that someone will complain they are downloading from somewhere weird and I will need a place to test to send them so I know what Continent/Country/Timezone they are getting.